tests/propulsion/letodar_2204_test.cpp

```diff
@@ -1,46 +1,56 @@
 #include "letodar_2204.hpp"
 #include <utility>
+#include "gmock/gmock.h"
 #include "gtest/gtest.h"
 
+using ::testing::NiceMock;
+using ::testing::Return;
+
 namespace {
 class Letodar2204Tests : public ::testing::Test {
  protected:
+  Letodar2204Tests() {}
   virtual void SetUp() {
-    unit_under_test_ = std::make_unique<propulsion::LeTodar2204>(std::make_unique<propulsion::Esc>(esc_));
+    ON_CALL(*esc_, GetMaxPulseDurationInMicroSeconds()).WillByDefault(Return(100));
+    ON_CALL(*esc_, GetMinPulseDurationInMicroSeconds()).WillByDefault(Return(10));
+    ON_CALL(*esc_, SetPulseDuration).WillByDefault(Return(types::DriverStatus::OK));
```

Comment:
For consistency you could remove '()' from GetMaxPulseDurationInMicroSeconds and GetMinPulseDurationInMicroSeconds. Or?

Reply:
You are correct.
Fixed with [this](https://github.com/Squiddrone/Squiddrone/pull/271/commits/0a5c06b2e31fe754073f4bf7810e6ee8f942a0d9) commit